projects/plugins/social/src/js/editor.js

```diff
@@ -120,6 +127,16 @@ const JetpackSocialSidebar = () => {
 				</PublicizePanel>
 			</PluginPrePublishPanel>
 
+			{ isSocialImageGeneratorEnabled && (
```

Comment:
I've only just realised that this adds it to Social, but not the core Jetpack plugin. I'm not sure if we would do that [here](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/extensions/plugins/publicize/index.js#L49) or [as a component here](https://github.com/Automattic/jetpack/blob/b313bd253afdf919adf7a71a866d0ab85b3a7bda/projects/plugins/jetpack/extensions/blocks/social-previews/index.js#L21)

Reply:
Let's hold off on worrying about bringing the feature to the Jetpack plugin. Prioritize building and making it available in the Social Plugin.